scripts/install-worker.sh

```diff
@@ -88,13 +88,19 @@ fi
 if cat /etc/*release | grep "al2023" > /dev/null 2>&1; then
   # exists in al2023 only (needed by kubelet)
   sudo yum install -y iptables-legacy
+
+  # Remove the amazon-ec2-net-utils package, if it's installed. This package interferes with the route setup on the instance.
+  if yum list installed | grep amazon-ec2-net-utils; then sudo yum remove amazon-ec2-net-utils -y -q; fi
```

Comment:
Is this still valid on AL2023 or is it just a carryover from AL2? Based on this statement in the project https://github.com/amazonlinux/amazon-ec2-net-utils#usage - have we reached out to the EC2 net utils project/team to figure out the proper solution instead of simply removing the package?

Reply:
@bryantbiggs not doing it idly :) been struggling with this one for 2 days now and finally figured out that the package name was different! https://testgrid.k8s.io/amazon-ec2#Conformance%20-%20EC2/EKS/AL2023%20-%20master&width=20

Reply:
This https://github.com/awslabs/amazon-eks-ami/pull/368#issuecomment-788417027 has the TL;DR on why `ec2-net-utils` is not compatible with aws-vpc-cni..

Reply:
@jayanthvn @bryantbiggs opened an issue on the other side: https://github.com/amazonlinux/amazon-ec2-net-utils/issues/100